Application/EdFi.Ods.Features/OpenApiMetadata/Providers/EnabledOpenApiMetadataDocumentProvider.cs

```diff
@@ -75,7 +75,8 @@ private string GetMetadataForContent(OpenApiContent content, HttpRequest request
             return content.Metadata
                 .Replace("%HOST%", Host())
                 .Replace("%TOKEN_URL%", TokenUrl())
-                .Replace("%BASE_PATH%", basePath);
+                .Replace("%BASE_PATH%", basePath)
+                .Replace("%SCHEME%", request.Scheme);
```

Comment:
should this be using our extension method to support reverse proxies?
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/007d47ddc3b7a17253320a1471e6de34fa90516a/Application/EdFi.Ods.Api/Extensions/HttpRequestExtensions.cs#L29-L47

Reply:
agreed.

Reply:
updated